### ISSUE TITLE
Speak throws exception in node REPL

### ISSUE THREAD

**Issue by wildeyes** [Loknar/node-winsay#3]
`set NODE_PATH=C:\Program Files\nodejs\node_modules`
then `node`

``` shell
> require('winsay').speak('sup')                                                                                       
undefined                                                                                                              
>                                                                                                                      
C:\...\v0.10.38\node_modules\winsay\winsay.js:25                                   
            throw error;                                                                                               
                  ^                                                                                                    
Error: Object reference not set to an instance of an object.                                                           
    at <anonymous>:1:55                                                                                                
    at speakSync (C:\...\v0.10.38\node_modules\winsay\winsay.js:22:5)              
    at process.<anonymous> (C:\...\v0.10.38\node_modules\winsay\speakAsync.js:4:5) 
    at process.emit (events.js:98:17)                                                                                  
    at handleMessage (child_process.js:322:10)                                                                         
    at Pipe.channel.onread (child_process.js:349:11)                                                                   

(^C again to quit)                                                                                                     
>                                                                                                                      
```

:( Why so?

**Comment by wildeyes** [comment]
Haven't looked at the usage example, works as expected now. Assumed it would work with the default voice if i don't specify a voice. Perhaps it should? :)

**Comment by Loknar** [comment]
my goal was to make a module that does the same and looks the same as [say.js](https://github.com/Marak/say.js) but runs on windows, does `say.js` support not specifying a voice and in that case use default voice?

edit:

Note: changing the voice hasn't been implemented yet (v0.0.4), hopefully someday I or someone else will find out how easy it is and implement it.

I understand that this makes the choose-voice variable completely useless, but maybe someday it'll be implemented :3
